modules/pam_mkhomedir/mkhomedir_helper.c

```diff
@@ -40,8 +40,10 @@ create_homedir(const struct passwd *pwd, mode_t dir_mode,
    int retval = PAM_SESSION_ERR;
 
    /* Create the new directory */
-   if (mkdir(dest, 0700) && errno != EEXIST)
+   if (mkdir(dest, 0700))
    {
+      if (errno = EEXIST)
```

Comment:
`errno = EEXIST` is an assignment, not a comparison.

Reply:
Fixed, thanks for noticing!